exercises/clock/example_gen.go

```diff
@@ -15,7 +16,8 @@ func main() {
 		log.Fatal(err)
 	}
 	var j js
-	if err := gen.Gen("clock.json", &j, t); err != nil {
+	f := path.Join("exercises", "clock", "canonical-data.json")
```

Comment:
(I know I kinda repeated myself and you already responded in the other PR, sorry about that)